### PR TITLE
Made note public variable

### LINKED ISSUE #42: Changes to note_off causing issues
Hi there.
I was trying to run the 'examples/midi_inoutdemo.py' script today and had a problem.
It seems the changes made by @dherrada in #33 to note_off.py - specifically changing the class variables to be preceded by an underscore (eg self.note is now self._note) mean that the demo crashes out at line 65 because 'NoteOff' has no attribute 'note'.
Why were these changes to note_off.py necessary? And is the best fix to revert the changes, add getter methods for 'note' etc, or change the example code?
I haven't managed to try it yet, but I'm a bit confused as to why this would not also cause an error when running 'tests/test_MIDI_unittests.py' since this looks like it would simulate this.

### PR DESCRIPTION
**Pull request by dherrada**
Fixes #42

### CODE REVIEW

File: adafruit_midi/note_off.py [hunk 1]
```diff
@@ -36,17 +36,17 @@ class NoteOff(MIDIMessage):  # pylint: disable=duplicate-code
     LENGTH = 3
 
     def __init__(self, note, velocity=0, *, channel=None):
-        self._note = note_parser(note)
+        self.note = note_parser(note)
```
Comment (tannewt) [inline adafruit_midi/note_off.py:39]: You may want to add a `"""comment"""` after `self.note` so that it gets documented.

**Review by tannewt — CHANGES_REQUESTED** [review]

File: adafruit_midi/note_off.py (hunk 1 above)
Comment (dherrada) [reply adafruit_midi/note_off.py:39]: Ah good idea. Will do.

File: adafruit_midi/note_off.py [hunk 2]
```diff
@@ -37,6 +37,7 @@ class NoteOff(MIDIMessage):  # pylint: disable=duplicate-code
 
     def __init__(self, note, velocity=0, *, channel=None):
         self.note = note_parser(note)
+        """ self.note: key, either int (0-127) or string that will be turned off """
```
Comment (tannewt) [inline adafruit_midi/note_off.py:40]: No need to add the prefix. It is determined by position.

```suggestion
        """key, either int (0-127) or string that will be turned off """
```

**Review by tannewt — CHANGES_REQUESTED** [review]

**Review by tannewt — APPROVED** [review]
Thanks!

**Pull request merged**